web/src/scripts/actions/elementTreeActions.js

```diff
@@ -47,6 +47,21 @@ export const selectElementFromPos = (filename, pos) => async (dispatch, getState
   return dispatch(deselectElement(filename))
 }
 
+export const selectElementByPath = (filename, elementPath) => async (dispatch, getState) => {
+  //TODO: Assumes the document is available and open, what if there is no document??
+  const {editor:{docCache}, elementTree: {elementTreeForFile}} = getState()
```

Comment:
basically the same as storyboards, we just need to `await dispatch(getDocument(path))`